Tests/test_image.py

```diff
@@ -94,7 +94,7 @@ def test_open_formats(self):
         with pytest.raises(TypeError):
             Image.open(PNGFILE, formats=123)
 
-        for formats in [["JPEG"], ("JPEG",)]:
+        for formats in [["jPeG"], ("JpEg",)]:
```

Comment:
```suggestion
        for formats in [["JPEG"], ("JPEG",), ["jpeg"], ["Jpeg"], ["jPeG"], ["JpEg"]]:
```
Just a suggestion.
My preference would be to keep the original test cases if possible.
I'm also adding in 'jpeg' and 'Jpeg', since I would think those are the most likely non-uppercase versions.